src/components/Form/types.ts

```diff
@@ -0,0 +1,41 @@
+export interface CurrentFormProps {
```

Comment:
if you have short interface for props, it is better to keep it in component file